rpc/src/services/context.rs

```diff
@@ -309,7 +323,8 @@ fn make_block_stats_impl(
         let action_stats = ActionStats {
             data: ActionData {
                 root: root.clone(),
-                actions_count: row.get(1)?,
+                tezedge_count: row.get(1)?,
+                irmin_count: row.get(20)?,
```

Comment:
@sebastiencs is the count ever different between the two?

Reply:
(other than when a single context is being used)

Reply:
@tizoc It's only different when a single context is used (one of the two will be zero)